app/models/host/discovered.rb

```diff
@@ -3,7 +3,7 @@
 class Host::Discovered < ::Host::Base
   include ScopedSearchExtensions
 
-  belongs_to :subnet
+  has_one :subnet, :through => :primary_interface
   belongs_to :hostgroup
```

Comment:
Hmmm but Host:Base does already have this:

    app/models/host/base.rb:25:    has_one :subnet, :through => :primary_interface

Why do we add this once again?